docs/blog/posts/announcing-gemini-tool-calling-support.md

```diff
@@ -0,0 +1,116 @@
+---
+draft: False
+date: 2024-09-03
+authors:
+  - ivanleomk
+---
+
+# Announcing Gemini and VertexAI Tool Calling Support
+
+We've excited to announce that Instructor now supports tool calling for both the Gemini SDK and the VertexAI SDk.
+
+A special shoutout to [Sonal](https://x.com/sonalsaldanha) for his contributions to the Gemini Tool Calling support.
+
+Let's walk through a simple example of how to use these new features
+
+## Installation
+
+To get started, install the latest version of Instructor. Depending on whether you're using Gemini or VertexAI, you should install the following:
+
+=== "Gemini"
+
+    ```bash
+    uv pip install "instructor[gemini]"
```

Comment:
why uv? its not the standard its just going t confuse average developers

Reply:
Ok reverted it back to just use pip